plugins/reporters/freemarker/src/main/kotlin/FreemarkerTemplateProcessor.kt

```diff
@@ -67,11 +67,6 @@ class FreemarkerTemplateProcessor(
     private val filePrefix: String = "",
     private val fileExtension: String = ""
 ) {
-    companion object {
-        const val OPTION_TEMPLATE_ID = "template.id"
-        const val OPTION_TEMPLATE_PATH = "template.path"
-    }
-
```

Comment:
How about squashing this to the commit where they became unused?

Reply:
I wanted to have the rather large change in the `asciidoc` plugin separate from this small cleanup in the `freemarker` plugin.